msteams-platform/messaging-extensions/how-to/micro-capabilities-for-website-links.md

```diff
@@ -0,0 +1,204 @@
+
+# [Education](#tab/education)
+
+|Property |Description  |
+|---------|---------|
+|`@type`     |   Course      |
+|image    | URL of the image of for the course.        |
+|name    |  Name of the course.       |
+|priceRange    |  Price of the course.       |
+|aggregateRating.ratingValue      |  Average rating of the course.       |
+|description    |  Summary about the course.       |
+|url     |  URL of the course website.       |
+|aggregateRating.reviewCount      | Average number of reviews for the course.        |
+|aggregateRating.ratingCount      | Average number of ratings for the course.        |
+
+Example of the unfurling experience for the course type:
+
+:::image type="content" source="../../assets/images/messaging-extension/micro-capabilities-template-course.png" alt-text="Screenshot shows the unfurling experience of the course template in Microsoft Teams.":::
+
+# [Restaurant](#tab/restaurant)
+
+|Property |Description  |
+|---------|---------|
+|`@type`     |  Restaurant       |
+|image    | URL of the image of for the restaurant.        |
+|name    |  Name of the author.       |
+|priceRange      |  Price range of the restaurant.       |
+|aggregateRating.ratingValue       | Average Rating of the restaurant.        |
+|address     |  Physical address of the restaurant.       |
+|url     | Link to the restaurant's website.        |
+|geo.latitude     | Latitude of the restaurant location.       |
+|geo.longitude      |  Longitude of the restaurant location.       |
+|aggregateRating.reviewCount      | Average number of reviews for the restaurant.        |
+|aggregateRating.ratingCount      | Average number of ratings for the restaurant.        |
+
+Example of the unfurling experience for the restaurant type:
+:::image type="content" source="../../assets/images/messaging-extension/micro-capabilities-template-restaurant.png" alt-text="Screenshot shows the unfurling experience of the restaurant template in Microsoft Teams.":::
+
+---
+
+The following is an example of schema metadata mapped to the rich unfurling preview of an Adaptive Card:
+
+:::image type="content" source="../../assets/images/messaging-extension/micro-capabilities-schema-map-actionable-previews.png" alt-text="Graphic shows an example of schema metadata mapped to elements in the rich unfurling preview of an Adaptive Card." lightbox="../../assets/images/messaging-extension/micro-capabilities-schema-map-actionable-previews.png":::
+
+### Validate your website link
+
+Go to [schema.org validator](https://validator.schema.org/) to validate if your website link metadata is as per schema.org standards. After the validation is successful, paste the website link in Teams compose message area to view rich unfurl preview of your link.
+
+:::image type="content" source="../../assets/images/messaging-extension/micro-capabilities-template-article.png" alt-text="Screenshot shows an example of rich unfurl preview experience of a micro-capability website link in Teams.":::
```

Comment:
This image is not the latest article preview as I mentioned above.

Reply:
Updated the article image as 
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/74c552db-df44-4876-92d3-8f74e2febd37)